extension/fts/src/function/query_fts_index.cpp

```diff
@@ -339,6 +339,7 @@ void QueryFTSAlgorithm::bind(const GDSBindInput& input, main::ClientContext& con
     auto graphEntry = graph::GraphEntry({termsEntry, docsEntry}, {appearsInEntry});
     bindData = std::make_unique<QueryFTSBindData>(std::move(graphEntry), nodeOutput,
         std::move(query), *ftsIndexEntry, QueryFTSConfig{input.optionalParams});
+    context.setUseInternalCatalogEntry(false /* useInternalCatalogEntry */);
```

Comment:
Now that I see this. I would double check CreateFTS and other places, e.g. create index & vector index. And make sure we always set to false afterwards.

Reply:
I don't think we support match query after standalone call function:
```
CALL CREATE_FTS_INDEX('doc', 'docIdx4', ['content', 'name']) match (n) return n
```